src/email/util/content.util.ts

```diff
@@ -0,0 +1,298 @@
+                    text-decoration: none;
+                  "
+                >
+                  Set Your Password
+                </a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        `
+  }
+
+  return `<div style="color:#000000">
+  Hi ${firstName} ${lastName},
+  <br />
+  <br />
+  It was great learning more about your campaign, and we're excited to help however we can.
+  <br />
+  <br />
+  To get started, please set up your password by clicking the button below. This will give you access to your dashboard, where you'll find all of our free tools including content creation, campaign tracker, and voter data.
+  <br />
+  <br />
+
+  <table
+      width="300"
+      style="
+        width: 300px;
+        background-color: #0D1528;
+        border-radius: 8px;
+      "
+      border="0"
+      cellspacing="0"
+      cellpadding="0"
+      align="center"
+    >
+      <tr>
+        <td
+          class="em_white"
+          height="42"
+          align="center"
+          valign="middle"
+          style="
+            font-family: Arial, sans-serif;
+            font-size: 16px;
+            color: #ffffff;
+            font-weight: bold;
+            height: 42px;
+          "
+        >
+          <a
+            href="${link}"
+            target="_blank"
+            style="
+              text-decoration: none;
+              color: #ffffff;
+              line-height: 42px;
+              display: block;
+            "
+          >
+            Set Your Password
+          </a>
+        </td>
+      </tr>
+    </table>
+    <br />
+    <br />
+
+    Also, we encourage you to share our endorsement of your campaign on social media using our share kit. Please follow this link to Canva to access the templates and receive instructions for accessing your endorsement image. Or, you can use this link for a quick video tutorial. Let us know if you have any questions about this, and please tag @goodpartyorg should you decide to post. We'll share it with our followers! 
+    <br />
+    <br /> 
+For more information about the offering and our organization, check out these links at your leisure:
+<ul>
+<li>An interactive demo of how to use GoodParty.org</li>
+<li>An overview of the benefits and our free campaigning tools</li>
+<li>More information about our mission and vision for empowering grassroots, independent candidates</li>
+</ul>
+If you have any questions about how to access the tool, our free SMS and yard signs offering, or would like to speak with one of our political associates, please let us know. We're thrilled to endorse your campaign and wish you the best of luck.
+    <br />
+    <br /> 
+All the best,
+    <br />
+    <br />
+GoodParty.org Team
+</div>
+  `
+}
+
+export function getBasicEmailContent(
+  msg = '',
+  // messageHeader was present in old tg-api version but not used
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  messageHeader = '',
```

Comment:
If this is no longer used, can we just omit it here then? What's the purpose of keeping this here w/ the comment?